xbmc/platform/darwin/ios/network/NetworkIOS.mm

```diff
@@ -0,0 +1,508 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include <cstdlib>
```

Comment:
clang-format didn't catch that one, because there is an empty line between and `IncludeBlocks: Regroup` isn't set in `.clang-format`.